modules/nodes-cluster-enabling-features-about.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes-cluster-enabling-features.adoc
```

Comment:
Should the full assembly path be here? `nodes/clusters/nodes-cluster-enabling-features.adoc`

Reply:
Good catch. I moved these files around a lot when converting from 3.11 to 4.x. I'm sure there are others just like this,